client/verta/tests/unit_tests/strategies.py

```diff
@@ -262,3 +260,98 @@ def mock_workspace(draw):
         )
     )
     return workspace
+
+
+@st.composite
+def mock_pipeline_definition(draw):
+    """Generate a mocked pipeline specification dictionary"""
+
+    # step names in a pipeline must be unique
+    step_names = draw(
+        st.lists(st.text(min_size=5, max_size=25), min_size=5, max_size=5, unique=True)
+    )
+    model_versions = draw(
+        st.lists(
+            st.text(alphabet=["1", "2", "3", "4", "5"], min_size=1),
+            min_size=5,
+            max_size=5,
+            unique=True,
+        )
+    )
+
+    return {
+        "graph": [
+            {"predecessors": [], "name": step_names[0]},
+            {"predecessors": [step_names[0]], "name": step_names[1]},
+            {"predecessors": [step_names[1]], "name": step_names[2]},
+            {"predecessors": [step_names[1]], "name": step_names[3]},
+            {"predecessors": [step_names[2], step_names[1]], "name": step_names[4]},
+        ],
+        "pipeline_version_id": draw(st.integers(min_value=1, max_value=1000)),
```

Comment:
Did you have a reason in mind for `max_value=1000`?

Reply:
Kept running into this error, so I tried setting some max limits as suggested:
![image](https://github.com/VertaAI/modeldb/assets/114943931/3939eebb-cb86-4e0d-a58d-0f9f6662e421)
Is there a better way of handling this?

Reply:
> . . . You could try setting `max_size` parameters on your collections . . .

That actually refers to setting `max_size` on strategies like `st.lists()` or `st.dictionaties()`.

`max_size` on `st.integers()` wouldn't affect that health check.

Reply:
Removed.  I'm only using max_value on integers now where it is required to avoid the protobuf "Value out of range" error on model version IDs.  Thanks